src/blockchain_processor.py

```diff
@@ -489,15 +480,13 @@ def do_unsubscribe(self, method, params, session):
                 if session in l:
                     print_log("error rc!!")
                     self.shared.stop()
-                if l == []:
-                    self.watched_addresses.pop(addr)
```

Comment:
@ecdsa let me know if this makes sense. As far as I could tell, this code is unreachable due to the `if not l: return` statement before it.
 `